src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -97,6 +104,14 @@
   }
 }
 
+.form-group {
+  p {
+    padding: 13px 13px 13px 0px;
+    color: #000000;
```

Comment:
We use variables for colors, so try to find an appropriate one and use it here

Reply:
done